src/pages/patientView/mutation/column/CohortColumnFormatter.tsx

```diff
@@ -5,52 +5,65 @@ import DefaultTooltip from 'shared/components/DefaultTooltip';
 import 'rc-tooltip/assets/bootstrap_white.css';
 import {MutationTableRowData} from "../../../../shared/components/mutationTable/IMutationTableProps";
 import {MutSigData} from "../../PatientViewPage";
-
-export interface IVariantCountData {
-    numberOfSamples?:number;
-    geneData?:{ [entrezGeneId:string]: {
-        numberOfSamplesWithMutationInGene?:number,
-        numberOfSamplesWithKeyword?:{ [keyword:string]:number }
-    }};
-};
+import {VariantCountCacheType} from "../../clinicalInformation/CohortVariantCountCache";
 
 export default class CohortColumnFormatter {
 
     public static renderFunction(data:IColumnFormatterData<MutationTableRowData>, columnProps:any) {
```

Comment:
done but it's useless for now because the types are not tracked at the point where columnProps is actually defined in `MutationInformationContainer.tsx`